ParisBNF/abb/BNFabb166.xml

```diff
@@ -0,0 +1,500 @@
+                                                                                                and between the columns, mostly in Gǝʿǝz and sometimes in Amharic</desc>
+                                                                                        </item>
+                                                                                        
+                                                                                        
+                                                                                        <item xml:id="e5">
+                                                                                            <locus target="#8v #9r #9v #12v #20v #21r #25r #26r #36r #65v #75r #78v #121r #132r"/>
+                                                                                            <desc type="Unclear"> Small crosses of unclear function drawn in the margins, 
+                                                                                                on these and some other folia</desc>
+                                                                                        </item>                                                                                        
+                                                                                                </list>
+                                                                                </additions>
+                                                </physDesc>
+                                    <history>
+                                        <origin>
+                                            <origDate notBefore="1700" notAfter="1799">18th century.</origDate>
+                                            The names of the commissioner and the scribe on <locus target="#149v"/>
+                                        have been erased. Only the second part of the scribe's name can be discerned, which seems to 
+                                        be <foreign xml:lang="gez">ʾIyasus</foreign> (Walda or Gabra ʾIyasus?).
+                                        </origin>
+                                    </history>
+                                </msPart>
+                                   
+                                  
+                    <msPart xml:id="p3">
+                        <msIdentifier/>
+                        <msContents>
+                            <summary/>
+                            
+                                <msItem xml:id="p3_i1">                                    
+                                    <locus from="150ra" to="150vc"/>                                    
+                                    <title type="incomplete" ref="LIT3171Commen#Homilies">Excerpt from Homily 1 and Admonition 1</title>            
```

Comment:
note: when you have text inside of the `title `tag (here: `Excerpt from Homily 1 and Admonition 1`) this will be the ONLY text printed. Neither the title of the main text nor the title of the label will be visible.

so you should either encode

`<title type="incomplete" ref="LIT3171Commen#Homilies">Commentary on the Epistle to the Hebrews: Excerpt from Homily 1 and Admonition 1</title>`

or `<title type="incomplete" ref="LIT3171Commen#Homilies"/>
<note>Excerpt from Homily 1 and Admonition 1</note>`

or

or have two msItems for `LIT3171Commen#Homily1` and `LIT3171Commen#Tagsas1`